### ISSUE TITLE
Additional D and I behaviors for errorOverrides to turn specific errors and warnings into debug or info messages

### ISSUE THREAD

**Issue by axelfontaine** [flyway/flyway#2212]
This adds the following new behaviors for `errorOverrides`:
- `D` to force a debug message
- `D-` to force a debug message, but do not show the original sql state and error code
- `I` to force an info message
- `I-` to force an info message, but do not show the original sql state and error code
- `W-` to force a warning, but do not show the original sql state and error code
- `E-` to force an error, but do not show the original sql state and error code

#### Usage example: SQL Server PRINT messages should not be shown as warnings

```sql
PRINT 'Starting ...';
PRINT 'Done.';
```

currently results in the following output:

```
WARNING: DB: Starting ... (SQL State: S0001 - Error Code: 0)
WARNING: DB: Done. (SQL State: S0001 - Error Code: 0)
```

but with `errorOverrides=S0001:0:I-` it becomes

```
Starting ...
Done.
```

**Comment by seb-urbaniak** [comment]
Thank you Axel.

**Comment by axelfontaine** [comment]
This has now been implemented and will be released in the next few days as part of 5.2.2.
